Makefile.am

```diff
@@ -53,15 +59,21 @@ stream_DATA = $(NULL)
 recordingsdir = $(datadir)/janus/recordings
 recordings_DATA = $(NULL)
 
+if WINDOWS_OS
+INSTDIR = ../..
```

Comment:
That doesn't entirely make sense ...

I regularly compile janus in `/home/scratch/janus-gateway` and install to `/opt/janus`. You mean, the user can change the install dir after compile time? That's already sort of possible, by specifying the config file location on the janus command line (and then the config file specifies all the other paths). But this doesn't remove the need to do that, you only use INSTDIR for the location to install the sample config files ...

Presumably, what you would want to do for a "windows installer", is specify minimal `--prefix`, `--confdir` etc, and use the existing `DESTDIR` feature to install to a temporary "package root" folder, then the installer can put that anywhere, and the last necessary step is to customize the config file and launcher shortcut, from the installer, if that's what you really want to do. (You might also want to enable janus to use a relative path to the main config file, and relative paths within it, but this doesn't do that either.)